src/lib/tooltip/tooltip.html

```diff
@@ -1,4 +1,6 @@
 <div class="mat-tooltip"
+     role="tooltip"
```

Comment:
[Looking at this example of an accessible tooltip](http://accessibility.athena-ict.com/aria/examples/tooltip.shtml), it looks like we need to toggle `aria-hidden`, in addition to the `role` and `aria-describedby`.